### PR TITLE
sidepanel and footer UI

### PR DESCRIPTION
**Pull request by jstsanam**
Problem statement: To make Side Navigation UI and Footer of the website

Preview:
image

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -1,7 +1,19 @@
...
+import SidePanel from "./components/side-panel/SidePanel";
+import HomeScreen from "./components/HomeScreen";
+import Footer from "./components/Footer";
 
 function App() {
-  return <Header />;
+  return (
+    <>
+      <Header />
+      <main className="flex gap-2" style={{ height: "29rem" }}>
```
Comment (som-matrix) [inline src/App.tsx:10]: no inline styles use tailwindcss please

File: src/components/Footer.tsx [hunk 2]
```diff
@@ -0,0 +1,24 @@
+export default function Footer() {
+  return (
+    <div
+      className="flex justify-between items-center m-2 pl-3 pr-3 pt-2 pb-2"
+      style={{
+        background: "linear-gradient(to right, #ae2996, #825fc3, #519af4)",
+      }}
```
Comment (som-matrix) [inline src/components/Footer.tsx:7]: no inline styles use tailwindcss please

File: src/components/HomeScreen.tsx [hunk 3]
```diff
@@ -0,0 +1,13 @@
+export default function HomeScreen() {
+  return (
+    <div
+      className="rounded-md mr-2 pl-4 pr-4 pt-3"
+      style={{
+        background: "linear-gradient(to bottom, #212121 10%, #121212 100%)",
+        width: "77%"
+      }}
```
Comment (som-matrix) [inline src/components/HomeScreen.tsx:8]: again no inline styles here use tailwind css please

File: src/components/side-panel/SidePanel.tsx [hunk 4]
```diff
@@ -0,0 +1,40 @@
+import React from "react";
+import SidePanelCards from "./SidePanelCards";
+import SidePanelFooter from "./SidePanelFooter";
+
+export default function SidePanel() {
+  return (
+    <div
+      className="rounded-md ml-2"
+      style={{ backgroundColor: "#121212", width: "23%" }}
```
Comment (som-matrix) [inline src/components/side-panel/SidePanel.tsx:9]: use tailwindcss

File: src/components/side-panel/SidePanelCards.tsx [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+  {
+    title: "Lt's find some podcasts to follow",
+    subtitle: "We'll keep you updated on new episodes",
+    buttonText: "Browse podcasts",
+  },
+];
+
+export default function SidePanelCards() {
+  return (
+    <div className="m-2 overflow-hidden hover:overflow-y-scroll" style={{height: "15rem"}}>
```
Comment (som-matrix) [inline src/components/side-panel/SidePanelCards.tsx:16]: remove inline styles , use tailwindcss

File: src/components/side-panel/SidePanelCards.tsx [hunk 6]
```diff
@@ -0,0 +1,35 @@
...
+      {cardsData.map((cardData: any) => (
+        <div className="p-3 mb-4 mt-2 rounded-md" style={{backgroundColor: "#1f1f1f"}}>
```
Comment (som-matrix) [inline src/components/side-panel/SidePanelCards.tsx:18]: remove inline styles , use tailwindcss

File: src/components/side-panel/SidePanelFooter.tsx [hunk 7]
```diff
@@ -0,0 +1,39 @@
...
+      "Cookies",
+      "About Ads",
+      "Accessibility"
+    ];
+  
+    return (
+      <div className="m-2 pl-3 pr-3 pt-6">
+        <ul>
+          {footerLinks.map((link, index) => (
+            <li key={index} className="text-gray-400 font-semibold inline-flex" style={{fontSize: "0.5em"}}>
```
Comment (som-matrix) [inline src/components/side-panel/SidePanelFooter.tsx:15]: remove inline styles , use tailwindcss

File: src/components/side-panel/SidePanelFooter.tsx [hunk 8]
```diff
@@ -0,0 +1,39 @@
...
+            className="mr-1"
+          >
+            <path d="M480-80q-82 0-155-31.5t-127.5-86Q143-252 111.5-325T80-480q0-83 31.5-155.5t86-127Q252-817 325-848.5T480-880q83 0 155.5 31.5t127 86q54.5 54.5 86 127T880-480q0 82-31.5 155t-86 127.5q-54.5 54.5-127 86T480-80Zm0-82q26-36 45-75t31-83H404q12 44 31 83t45 75Zm-104-16q-18-33-31.5-68.5T322-320H204q29 50 72.5 87t99.5 55Zm208 0q56-18 99.5-55t72.5-87H638q-9 38-22.5 73.5T584-178ZM170-400h136q-3-20-4.5-39.5T300-480q0-21 1.5-40.5T306-560H170q-5 20-7.5 39.5T160-480q0 21 2.5 40.5T170-400Zm216 0h188q3-20 4.5-39.5T580-480q0-21-1.5-40.5T574-560H386q-3 20-4.5 39.5T380-480q0 21 1.5 40.5T386-400Zm268 0h136q5-20 7.5-39.5T800-480q0-21-2.5-40.5T790-560H654q3 20 4.5 39.5T660-480q0 21-1.5 40.5T654-400Zm-16-240h118q-29-50-72.5-87T584-782q18 33 31.5 68.5T638-640Zm-234 0h152q-12-44-31-83t-45-75q-26 36-45 75t-31 83Zm-200 0h118q9-38 22.5-73.5T376-782q-56 18-99.5 55T204-640Z" />
+          </svg>
+          English
+        </button>
+      </div>
+    );
+  }
+
```
Comment (som-matrix) [inline src/components/side-panel/SidePanelFooter.tsx:39]: add new line

File: src/components/Footer.tsx [hunk 9]
```diff
@@ -0,0 +1,24 @@
...
+      </div>
+      <button
+        type="button"
+        className="text-gray-900 bg-gray-100 hover:bg-gray-200 font-bold rounded-full text-sm px-5 py-2.5 text-center inline-flex items-center"
+      >
+        Sign up free
+      </button>
+    </div>
+  );
+}
```
Comment (som-matrix) [inline src/components/Footer.tsx:24]: image

Fix the footer

File: src/components/side-panel/SidePanelFooter.tsx (hunk 8 above)
Comment (jstsanam) [reply src/components/side-panel/SidePanelFooter.tsx:39]: didn't get, can you please elaborate more?

File: src/components/Footer.tsx (hunk 9 above)
Comment (jstsanam) [reply src/components/Footer.tsx:24]: Okay

**Pull request merged**